package.json

```diff
@@ -55,6 +55,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-redux": "^7.1.3",
+    "react-router": "^5.1.2",
```

Comment:
just curious, what is the use of `react-router` because I see we're using `react-router-dom`

Reply:
react-router-dom is a wrapper of react-router that has some additional functionalities to be used in web dev. It's not compulsory to install the react-router library by itself. let me remove it